control-plane/grpc/src/operations/node/client.rs

```diff
@@ -84,4 +84,34 @@ impl NodeOperations for NodeClient {
             None => Err(ReplyError::invalid_response(ResourceKind::Block)),
         }
     }
+
+    async fn cordon(&self, id: NodeId, label: String) -> Result<Node, ReplyError> {
```

Comment:
Do we need to add the tracing tags here? ref: https://github.com/openebs/mayastor-control-plane/blob/08f07f949db2a852a13ea98bc56ee457f8ca222f/control-plane/grpc/src/operations/node/client.rs#L43